helm/api-testing/values.yaml

```diff
@@ -5,10 +5,16 @@
 replicaCount: 1
 
 image:
+  registry: docker.io
```

Comment:
I prefer to use `linuxsuren.docker.scarf.sh` as the default registry. It is a mirror of `docker.io`. Because it could collect more  information about the pulling data.